KPI/H.adoc

```diff
@@ -1,235 +1,239 @@
 == H. Keywords
 
```

Comment:
Thanks for changing the formatting, which makes it very readable now and hopefully clear enough for whoever will implement.
Perhaps to share with the rest of the group the format template for KPI's.
Who is the intended audience for the KPI deliverable?
Is it just for developers so they can implement the rules or for a wider audience?  
Maybe for discussion in one of the issues instead.

Reply:
Thanks for the feedback @martaguti.  We can cover and document this in #69.